rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py

```diff
@@ -0,0 +1,29 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_primary_secondary_loops_dict import (
+    get_primary_secondary_loops_dict,
+)
+from rct229.utils.jsonpath_utils import find_all, find_one
+
+
+def get_heat_rejection_loops_connected_to_baseline_systems(rmi):
+    """
+    Get a list of all heat rejection loops in an RMI that are connected to a baseline HVAC System (Type-7, 8, 11.1,
+    11.2, 12, 13, 7b, 8b, 11.1b, 12b)
+
+    Parameters
+    ----------
+    rmi dict
+        A dictionary representing a ruleset model instance as defined by the ASHRAE229 schema
+
+    Returns -------
+    heat_reject_loop_list list
+        A list saves the ids of heat rejection loops in the model that serve baseline system types. If no qualifying
+        heat rejection loops, return an empty list.
+    """
+    primary_secondary_loops_dict = get_primary_secondary_loops_dict(rmi)
+    heat_rejection_loop_list = [
+        chiller["condensing_loop"]
```

Comment:
I think this should be `chiller["condinsing_loop"]["id"]`.

Reply:
The schema seems to be a reference to a [fluid loop](https://github.com/open229/ruleset-model-description-schema/blob/main/docs229/ASHRAE229.schema.json#L3356-L3360)

Reply:
Yep. Your are correct.